src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,51 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final String PACKAGE_NAME = "mate.jdbc";
+
     public static void main(String[] args) {
-        // test your code here
+        Injector injector = Injector.getInstance(PACKAGE_NAME);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+
+        Driver bombilla = driverService.create(new Driver(null, "Bombilla", "131313"));
+        Driver lazyMan = driverService.create(new Driver(null, "Lazy Man", "6"));
+        Driver veteran = driverService.create(new Driver(null, "Veteran", "000001"));
+        Driver lucky = driverService.create(new Driver(null, "Lucky", "7777777"));
+
+        Manufacturer mersProducer = manufacturerService
+                .create(new Manufacturer(null, "Daimler AG", "Germany"));
+        Manufacturer audiProducer = manufacturerService
```

Comment:
taking into account, that we call model Manufacturer, it's better to use Manufacturer in variables names, not producer